src/nmodl/nocpout.cpp

```diff
@@ -1886,8 +1871,8 @@ static void declare_p() {
 	}
 }
 
-List *set_ion_variables(block)
-	int block;	/* 0 means equation block , 2 means initial block */
+List *set_ion_variables(int block)
+/* 0 means equation block , 2 means initial block */
```

Comment:
would it be a huge pain to move those into doxygen style since that's anyway something to do?

Reply:
I would not spend time on it, since the subsequent goal is refactor. That can be accomplished there